worker/setup.py

```diff
@@ -84,11 +84,6 @@ def make_release_tree(self, base_dir, files):
         'Topic :: Software Development :: Build Tools',
         'Topic :: Software Development :: Testing',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
```

Comment:
Buildbot worker should still support the oldest Python release possible. People run worker on all kinds of weird and old machines.